website/docs/basics/blocks-faq.md

```diff
@@ -50,11 +50,20 @@ The answer is the same as the previous one. In the global `manifest.json`, we ha
 
 ### If I want to create a new block/component, what do I do?
 
-In your terminal, write `wp boilerplate use_block --name=example` and style that block however you seem fit.
+In your terminal, write
+``` shell
+wp boilerplate use_block --name=example
+```
+and style that block however you seem fit.
```

Comment:
Maybe we should add here a note that if the block uses a component part for editor/option/toolbar, they should use the `use_component` command as well. 

Because they will get errors about missing component parts.

Reply:
agreed

Reply:
added